week_6/src/main/scala/org/epfl/errorhandling/part9/resilientGetAllPages.scala

```diff
@@ -12,28 +12,27 @@ def getPage(page: Int): Future[String] =
     Future.failed(Exception(s"Timeout when fetching page $page"))
   else Future(s"Page $page")
 
-def getAllPages(): Future[Seq[String]] =
-  getPagesCount().flatMap { totalPages =>
-    Future.traverse(1 to totalPages)(getPage)
-  }
-
-def resilientGetAllPages(): Future[Seq[String]] =
-
+def resilientGetPage(page: Int): Future[String]
```

Comment:
This doesn't compile because of a missing `=` sign

```suggestion
def resilientGetPage(page: Int): Future[String] =
```